<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>661</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>661</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>661</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="usename_layout" stretch="3,9">
    <item>
     <widget class="QLabel" name="username_lb">
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="username_le">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="pwd_layout" stretch="3,9">
    <item>
     <widget class="QLabel" name="pwd_lb">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pwd_le">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="checkpwd_layout" stretch="3,9">
    <item>
     <widget class="QLabel" name="checkpwd_lb">
      <property name="text">
       <string>确认密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="checkpwd_le">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttons_layout">
    <item>
     <widget class="QPushButton" name="submit_btn">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,9">
    <item>
     <widget class="QLabel" name="ques_lb">
      <property name="text">
       <string>密保问题</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ques_le"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,9">
    <item>
     <widget class="QLabel" name="ans_lb">
      <property name="text">
       <string>密保答案</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ans_le"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>Register</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>butt()</slot>
 </slots>
</ui>
